lib/elixir/lib/file.ex

```diff
@@ -361,6 +361,39 @@ defmodule File do
   end
 
   @doc """
+  Reads the symbolic link at `path`. If path exists and is a symlink, returns `{:ok, target}`,
```

Comment:
Please break `If path exists...` into a new line to keep summaries short (summaries are used in docs).